src/ansible_compat/runtime.py

```diff
@@ -346,24 +347,15 @@ def _ensure_module_available(self) -> None:
                 _AnsibleCollectionFinder,  # noqa: PLC2701
             )
 
-            if self.version >= Version("2.15.0.dev0"):
-                # pylint: disable=import-outside-toplevel,no-name-in-module
-                from ansible.plugins.loader import init_plugin_loader
-
-                _AnsibleCollectionFinder(  # noqa: SLF001
-                    paths=col_path,
-                )._remove()  # pylint: disable=protected-access
-                init_plugin_loader(col_path)
-            else:
-                # noinspection PyProtectedMember
-                # pylint: disable=protected-access
-                col_path += self.config.collections_paths
-                col_path += os.path.dirname(  # noqa: PTH120
-                    os.environ.get(ansible_collections_path(), "."),
-                ).split(":")
-                _AnsibleCollectionFinder(  # noqa: SLF001
-                    paths=col_path,
-                )._install()  # pylint: disable=protected-access
+            # noinspection PyProtectedMember
+            # pylint: disable=protected-access
+            col_path += self.config.collections_paths
+            col_path += os.path.dirname(  # noqa: PTH120
+                os.environ.get(ansible_collections_path(), "."),
+            ).split(":")
+            _AnsibleCollectionFinder(  # noqa: SLF001
+                paths=col_path,
+            )._install()  # pylint: disable=protected-access
```

Comment:
Not sure if I'm missing something, but this seems to be retaining the `< 2.15.0.dev0` path?

Reply:
Oops, read this after I merged it. Next time add -1. I am making a patch now. Second question would be.... why testing did not find any issues.

Reply:
Please review https://github.com/ansible/ansible-compat/pull/444

And, thanks for spotting it!!!